shared/src/business/useCases/generatePractitionerCaseListPdfInteractor.test.ts

```diff
@@ -16,6 +16,11 @@ describe('generatePractitionerCaseListPdfInteractor', () => {
   });
 
   it('returns an unauthorized error on non internal users', async () => {
+    applicationContext
+      .getUseCases()
+      .getPractitionerCasesInteractor.mockImplementation(() => {
+        throw new Error('Unauthorized');
+      });
```

Comment:
The unauthorized error is now thrown on `getPractitionerCasesInteractor`. That said, I think it is still important to test that `generatePractitionerCaseListPdfInteractor` independently "bubbles up" that error.